src/Stratis.Bitcoin.IntegrationTests/Wallet/ColdWalletTests.cs

```diff
@@ -103,6 +103,7 @@ private CoreNode CreatePowPosMiningNode(NodeBuilder nodeBuilder, Network network
         /// Success is determined by whether the balance in the cold wallet increases.
         /// </description>
         [Fact]
+        [Trait("Unstable", "True")]
```

Comment:
@noescape00 you are looking at coldstake please remember to check this test as well